sdk/eventhub/service.projects

```diff
@@ -7,13 +7,18 @@
   the set of projects to to match what type of SDK we are interested in building.
 
   SDKType values
-   - empty or "client" will build the new track 2 client/data libraries
+   - "all" will build all projects
```

Comment:
hrmmm.... since you pushed the "default to all" up into the services project, empty will now result in nothing.   Should we default that again, or validate and trigger an error?

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/19860/files#diff-d111a8b1c659487619cd069817a1709ec942c379187c83b0e8a878f32800bc6eR3 we default empty to `all`.

Reply:
My bad.  I overlooked that.